containers/Dockerfile.buildenv

```diff
@@ -154,6 +154,23 @@ RUN wget https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.7.tar.gz -O bla
     && cd ../.. \
     && rm -rf yaml-cpp*
 
+# Install include-what-you-use
+RUN apt-get install -y libclang-10-dev
```

Comment:
The docker documentation recommends always running `apt-get update -y` before an `apt-get install`. The reason is that docker caches layers, so you can't rely on an earlier `apt-get update`.